examples/model_selection/plot_roc.py

```diff
@@ -19,44 +19,32 @@
 -------------------
 
 ROC curves are typically used in binary classification to study the output of
-a classifier. In order to extend ROC curve and ROC area to multi-class
-or multi-label classification, it is necessary to binarize the output. One ROC
-curve can be drawn per label, but one can also draw a ROC curve by considering
-each element of the label indicator matrix as a binary prediction
-(micro-averaging).
-
-Another evaluation measure for multi-class classification is
-macro-averaging, which gives equal weight to the classification of each
-label.
+a classifier. The ROC curve and ROC area can be extended to multi-class or
+multi-label classification by using the One-vs-Rest or One-vs-One scheme.
 
 .. note::
 
     See also :func:`sklearn.metrics.roc_auc_score`,
              :ref:`sphx_glr_auto_examples_model_selection_plot_roc_crossval.py`.
-
 """
-print(__doc__)
```

Comment:
don't remove this line

Reply:
Why do we print out the docstring for our `bench_*` and `plot_*` files? (I always found this a little weird)

Reply:
I guess it's just so you can run a script and have a description of what it does without having to open the file.